scripts/bench-bencher.sh

```
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+# Enhanced benchmark script with Bencher.dev integration
+
+set -euo pipefail
+
+# Load environment variables if .env exists
+if [ -f .env ]; then
+    export $(grep -v '^#' .env | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)